modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/behavior/impl/MailActivityBehavior.java

```diff
@@ -183,11 +184,38 @@ protected MultiPartEmail createMultiPartEmail(String text) {
             throw new FlowableException("Could not create text-only email", e);
         }
     }
+    
+    protected void validateToCcBcc(CommandContext commandContext, String to, String cc, String bcc, String tenantId) {
+        
+        if(to == null && cc == null && bcc == null) {
+            throw new FlowableException("No recipient could be found for sending email");
+        }
+        
+        String newTo, newCc, newBcc;
+        newTo = newCc = newBcc = getForceTo(commandContext, tenantId);
+        if (newTo == null) {
+            newTo = to;
+        }
+        
+        String[] tos = splitAndTrim(newTo);
+        if (newCc == null) {
+            newCc = cc;
+        }
+        
+        String[] ccs = splitAndTrim(newCc);
+        if (newBcc == null) {
+            newBcc = bcc;
+        }
+        
+        String[] bccs = splitAndTrim(newBcc);
+        if(tos == null && ccs == null && bccs == null) {
```

Comment:
This is only a style comment: please use `if (` instead of `if(`.  You have both versions in this PR.   Thanks

Reply:
@dbmalkovsky , I did above styling changes in both MailActivityBehavior files. Thanks.